conu/utils/filesystem.py

```diff
@@ -39,6 +46,10 @@ def __init__(self, path, mode=None, facl_rules=None, selinux_context=None, selin
             if get_selinux_status() == "Disabled":
                 raise ConuException("You are trying to apply SELinux labels, but SELinux is "
                                     "disabled on this system. Please enable it first.")
+
+        # if set to True, it means the directory is created and set up
+        self._initialized = False
```

Comment:
Attributes are being changed also when already existing directories are specified in path. Is this intended?

Reply:
From my PoV, yes. That's why I made sure that `os.makedirs` won't fail if the directory already exists.

Do you think this is bad?

Reply:
Yes, it extends usability of the class. I was just wondering if it wouldn't cause some problems, e.g when user unintentionally provides wrong path - relative path to some unrelated file. But apparently an edge-case, so approving :)

Reply:
so I really hope that no one will do path=/